src/main/java/iggly/Duke.java

```diff
@@ -0,0 +1,103 @@
+package iggly;
+
+import java.util.Scanner;
+
+import iggly.command.Command;
+import iggly.command.ExitCommand;
+import iggly.command.GreetCommand;
+import iggly.duke.DukeException;
+import iggly.duke.Parser;
+import iggly.duke.Storage;
+import iggly.javafx.Launcher;
+import iggly.model.TaskList;
+
+/**
+ * {@link Duke} is the main class of this program.
+ *
+ * @author Tsui Yi Wern (yiwern5)
+ */
+public class Duke {
+    private final TaskList taskList;
+    private final Storage storage;
+    private boolean isExit;
+
+    /**
+     * Constructor for {@link Duke} to initialize the storage or
+     * get task list from the storage.
```

Comment:
Should there a bit an empty line here to separate the parameter and description to improve readability?

Reply:
Alright! Will add a line between parameter and description, thanks for the suggestion :)